util/validators/jsonSchema/rest.ts

```diff
@@ -0,0 +1,207 @@
+import { Housing, Kita, Post, Refugee } from 'util/interfaces/rest';
+import { JSONSchemaType } from 'ajv';
+
+export const NewHousingREST: JSONSchemaType<Housing> = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Housing',
+  description: 'Data for a new Housing',
+  type: 'object',
+  properties: {
+    housing: {
+      type: 'object',
+      properties: {
+        housing_type: {
+          type: 'string',
+          minLength: 1,
+        },
+        people_limit: {
+          type: 'integer',
+          minimum: 1,
+        },
+        people_assigned: {
+          type: 'integer',
+          minimum: 0,
+          nullable: true,
+        },
+        size: {
+          type: 'number',
+          minimum: 1,
+          maximum: 300,
+        },
+        shared_bathroom: {
+          type: 'boolean',
+          nullable: true,
+        },
+        rooms: {
+          type: 'number',
+          minimum: 1,
+          maximum: 15,
+          nullable: true,
+        },
+        rent: {
+          type: 'number',
+          minimum: 0,
+          maximum: 3000,
+        },
+        info: {
+          type: 'string',
+          minLength: 0,
+          nullable: true,
+        },
+        citizen_id: {
+          type: 'number',
+          minimum: 1,
+        },
+      },
+      required: ['housing_type', 'size', 'rent', 'people_limit', 'citizen_id'],
+      additionalProperties: false,
+    },
+    address: {
+      type: 'object',
+      properties: {
+        house_number: {
+          type: 'integer',
+          minimum: 1,
+          maximum: 1000,
+        },
+        street: {
+          type: 'string',
+          minLength: 1,
+        },
+        city_code: {
+          type: 'integer',
+          minimum: 1,
+          maximum: 99999,
+        },
+      },
+      required: ['house_number', 'street', 'city_code'],
+      additionalProperties: false,
+    },
+  },
+  required: ['housing', 'address'],
+  additionalProperties: false,
+};
+
+export const KitaREST: JSONSchemaType<Kita> = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Kita Application',
+  description: 'Data for a new Kita Application',
+  type: 'object',
+  properties: {
+    care_time: {
+      type: 'number',
+      minimum: 20,
+      maximum: 45,
+    },
+    child: {
+      type: 'object',
+      properties: {
+        citizen_id: {
+          type: 'number',
+          minimum: 1,
+        },
+        refugee_id: {
```

Comment:
In der Spezifikation taucht nur die citizen_id auf. Hier wird aber auch die refugee_id benötigt. Auch hier das einmal einheitlich gestalten